03week/towersOfHanoi.js

```diff
@@ -19,24 +19,82 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
-
+// moves the piece from the first stack to the second stack
+const movePiece = (input1, input2) => {
+  return stacks[input2].push(stacks[input1].pop(input1));
 }
 
-function isLegal() {
-  // Your code here
+// return true if both inputs (combined into an array) contain only expected values ['a', 'b', 'c']
+// Note: values have already been converted to lowercase and trimmed of spaces
+const isValid = (input1, input2) => {
+  if (input1 && input2) {
+    const combinedInputs = input1 += input2;
```

Comment:
why are you reassigning what input1 is? best practice would be to make a new variable